.github/workflows/pr-comments.yml

```diff
@@ -39,16 +40,15 @@ jobs:
       - name: Build Hue
         run: |
           sudo apt-get update
-          sudo apt-get install -y gcc g++ build-essential python3.11-dev python3.11-venv python3.11-distutils asciidoc rsync curl sudo libkrb5-dev libldap2-dev libsasl2-dev libxml2-dev libxslt-dev  libsasl2-modules-gssapi-mit libsnappy-dev libffi-dev # This should not be needed as some point
-          sudo curl -sL https://bootstrap.pypa.io/get-pip.py | sudo python3.11
-          sudo apt-get install -y python3-setuptools
-          sudo apt-get install -y libncursesw5-dev libgdbm-dev libc6-dev libssl-dev openssl
+          sudo apt-get install -y build-essential asciidoc libkrb5-dev libldap2-dev libsasl2-dev libxslt-dev libsasl2-modules-gssapi-mit libsnappy-dev libgdbm-dev
 
           export PYTHON_VER=python3.11
+          export SKIP_PYTHONDEV_CHECK=true
           export ROOT=$PWD
           make apps test_prep
 
       - name: Run python unit tests
+        continue-on-error: true
```

Comment:
If some unit tests fail, how can we detect that? Will the workflow pass even if some tests fail?